src/object/object_primitive.c

```diff
@@ -9056,15 +9056,18 @@ pr_midxkey_add_prefix (DB_VALUE * result, DB_VALUE * prefix, DB_VALUE * postfix,
 #if !defined(NDEBUG)
   for (i = 0; i < n_prefix; i++)
     {
+      /* Compressed columns must be null. */
       assert (or_multi_is_null (midx_postfix->buf, i));
     }
 #endif
 
+  i = n_prefix;
```

Comment:
How about meaningful name instead of 'i'

Reply:
This is the index of the column currently being accessed.
`col_index` can be used instead of `i`.
But the function is not long. Isn't it okay to use `i`?